http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/ReactivecomplexClientResource.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+@Path("/clients/{rootPath}/resource-server")
+public class ReactivecomplexClientResource {
+    @Path("/{id}")
+    @DELETE
+    public String deleteById(@PathParam("rootPath") String rootPath, @PathParam("id") String id) {
+        return "/clients/" + rootPath + "/resource-server/" + id;
```

Comment:
Returning an URl to Delete action is slightly confusing and slightly against RFC[1]. Since we do not check full path in the tests, maybe we should replace it with "sent from resource-server" or something like that??

[1] https://httpwg.org/specs/rfc9110.html#DELETE

Reply:
We still return URI. Why is it better, than, say, String?

Reply:
Well, the issue is that the Http reactive client overwrites a sub-resource path param with a parent pathParam. So, I think that makes sense to return the full path just to verify that the "id" is not overwritten by the rootpath.